cpp/oneapi/dal/table/backend/convert/copy_convert_impl_dpc.cpp

```diff
@@ -13,7 +13,7 @@
 * See the License for the specific language governing permissions and
 * limitations under the License.
 *******************************************************************************/
-
+#include <iostream>
```

Comment:
Does it need to include iostream here?

Reply:
I guess it was for the debug, please remove it